src/server/core.js

```diff
@@ -11,23 +11,51 @@ function initTopics(topics) {
 
 function vote(state, topicId, voterId) {
   const voter = state.voters.find(voter => voter.id === voterId);
+
   if (!voter) {
-    state.topics[topicId].votes++;
-    state.voters.push({ id: voterId, vote: topicId });
+    const topics = Object.assign({}, state.topics, {
```

Comment:
we could use already declared in upper scope variables topics/voters to not to declare them twice

Reply:
Not sure about this one, what variables do you referring?
I don't see variables with names `topics`/`voters` in upper scope...

Reply:
I mean making variables at the top of the function if we using it twice, but now it's ok, nevermind